Examples/Python/src/Svg.cpp

```diff
@@ -42,40 +42,75 @@ using namespace ActsExamples;
 
 namespace {
 
-using ViewAndRange = std::tuple<std::string, Acts::Extent>;
-
-// Helper function to be picked in different access patterns
+// A cache object
+using PortalCache = std::list<std::string>;
```

Comment:
For my own curiosity, why is this a list? Does the portal selection need to observe any king of sorting?

Reply:
should be a `set` - will change in a future PR.